Philip_Cron/assignments/MathDojo.rb

```diff
@@ -0,0 +1,25 @@
+class MathDojo
+    attr_accessor :result
+    def initialize
+        @result = 0
+    end
+    def add(*val)
+        if val.class == Array
+            @result += val.flatten.reduce(:+)
+        else
+            @result += val
```

Comment:
you could actually skip the whole if/else part and just make it one line

@result += val.flatten.reduce(:+)   Even if that val isn't an array it'll just return as already flattened and add to the result